site/content/troubleshooting-guide.md

```diff
@@ -1 +1,122 @@
+|------------------------------------------------------------|----------------------------------------------------------------------------------------|----------------------------------------------|-----------------|
+| ASP.NET Core App to Amazon ECS using Fargate               | Microsoft.NET.Sdk.Web                                                                  | netcoreapp2.1, netcoreapp3.1, net5.0, net6.0 | Yes             |
+| ASP.NET Core App to AWS App Runner                         | Microsoft.NET.Sdk.Web                                                                  | netcoreapp2.1, netcoreapp3.1, net5.0, net6.0 | Yes             |
+| Service on Amazon ECS using Fargate                        | Microsoft.NET.Sdk                                                                      | netcoreapp2.1, netcoreapp3.1, net5.0, net6.0 | Yes             |
+| Scheduled Task on Amazon ECS using Fargate                 | Microsoft.NET.Sdk                                                                      | netcoreapp2.1, netcoreapp3.1, net5.0, net6.0 | Yes             |
+| Blazor WebAssembly App                                     | Microsoft.NET.Sdk.BlazorWebAssembly (for.NET 5.0+) Microsoft.NET.Sdk.Web (for.NET 3.1) | netstandard2.1                               | No              |
+| ASP.NET Core App to AWS Elastic Beanstalk on Linux         | Microsoft.NET.Sdk.Web                                                                  | netcoreapp2.1, netcoreapp3.1, net5.0, net6.0 | No              |
+| Push Container Images to Amazon Elastic Container Registry | N/A                                                                                    | N/A                                          | Yes             |
+
+You need to ensure that your .NET application falls into one of the categories in order to leverage AWS.Deploy.Tools
+
+Another reason why there are no recommendations generated is if your application `.csproj` file is using a variable for the `TargetFramework` property.
+
+
+    <Project Sdk="Microsoft.NET.Sdk.Web">
+	    <PropertyGroup>
+		    <TargetFrameworkVersion>net5.0</TargetFrameworkVersion>
+		    <TargetFramework>$(TargetFrameworkVersion)</TargetFramework>
+	    </PropertyGroup>
+	</Project>
+    
+No recommendations will be generated for the above `.csproj` file.
+**This is a bug which we will address.** Meanwhile, provide explicit values for the `TargetFramework` property.
+
+## Failed to create zip archive of the application
+Non-container based deployments types (such as deployments to AWS Elastic Beanstalk) create a zip file of the artifacts produced by the `dotnet publish` command.
+
+The zip command line utility is not installed by default on **Linux** based operating systems. If you are deploying using a non-container based option and encounter an error saying `failed to create a deployment bundle`, it is likely that zip is not installed on your system.
```

Comment:
Change "not installed by default on **Linux** based operating systems" to "not installed by default on SOME **Linux** distributions".

Reply:
Do we really not give a better error message if zip is created then "failed to create a deployment bundle"? If so we need to fix that. The error message should say something like "Unable to find the zip utility used to zip the output of the `dotnet publish` command".

Reply:
Also it would be good to explain to the user we use the zip utility to maintain Linux file permissions. Many people ask why we don't just use the built in .NET zip APIs.